src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -168,7 +176,9 @@ internal static async Task ConfigurePcs(
         builder.AddDependencyFlowProcessors();
         builder.AddVmrRegistrations(gitHubToken);
         builder.AddMaestroApiClient(managedIdentityId);
-        builder.AddGitHubClientFactory();
+        builder.AddGitHubClientFactory(
+        builder.Configuration.GetRequiredValue(ConfigurationKeys.GitHubClientId),
```

Comment:
This and the line below should probably be spaced out a bit
```suggestion
            builder.Configuration.GetRequiredValue(ConfigurationKeys.GitHubClientId),
```

Reply:
Done